test/unit/test-custom-element.js

```diff
@@ -2000,16 +2000,17 @@ describes.realWin('CustomElement', {amp: true}, (env) => {
         expect(toggle).to.have.not.been.called;
       });
 
-      it('should start loading when measured if already in viewport', () => {
-        stubInA4A(false);
-        const toggle = env.sandbox.spy(element, 'toggleLoading');
-        element.isInViewport_ = true;
-        element.setAttribute('layout', 'fill');
-        container.appendChild(element);
-        element.updateLayoutBox({top: 0, width: 300});
-        expect(toggle).to.be.calledOnce;
-        expect(toggle.firstCall.args[0]).to.equal(true);
-      });
+      // WHY when measured?
+      // it('should start loading when measured if already in viewport', () => {
```

Comment:
What's going on with this test?

Reply:
I need to either remove the test or modify it to check the inverse since I'm changing the behavior under test.